src/components/molecules/RegisterHeader/style.ts

```diff
@@ -0,0 +1,39 @@
+import Styled from 'styled-components';
+
+interface ISHeader {
+  url?: string;
+}
+
+const SHeader = Styled.div<ISHeader>`
```

Comment:
```suggestion
export const SHeader = Styled.div<ISHeader>`
```
나중엔 이렇게 선언하는게 더 나을거에요!

Reply:
넵 스타일부분은 전체적으로 건들여야해서 한번에 고쳐서 올리겠습니다~